operations/node_management/Reboot_NCNs.md

```diff
@@ -338,9 +345,9 @@ Before rebooting NCNs:
 
        If the configurationStatus is `pending`, wait for the job to finish before continuing. If the configurationStatus is `failed`, this means the failed CFS job configurationStatus should be addressed now for this node. If the configurationStatus is `unconfigured` and the NCN personalization procedure has not been done as part of an install yet, this can be ignored.
```

Comment:
```suggestion
       If the `configurationStatus` is `pending`, then wait for the job to finish before continuing. If the `configurationStatus` is `failed`, then the unsuccessful CFS job should be addressed now for this node. If the `configurationStatus` is `unconfigured` and the NCN personalization procedure has not been done as part of an install yet, then this can be ignored.
```